code/client/clrcore/External/VehicleHash.cs

```diff
@@ -1,481 +1,526 @@
-namespace CitizenFX.Core
+﻿namespace CitizenFX.Core
 {
-	public enum VehicleHash : uint
-	{
```

Comment:
Why is this entire file modified? Are you using `core.autocrlf=true` and did editorconfig get read?